### PR TITLE
Jobs Client API Design

### PR DESCRIPTION
**Pull request by barustum**
Will review with the team during the API Design meeting.

### CODE REVIEW

File: sdk/iot/Azure.Iot.Hub.Service/README.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+## Examples
+
+Include code snippets and short descriptions for each task you listed in the [Introduction](#introduction) (the bulleted list). Briefly explain each operation, but include enough clarity to explain complex or otherwise tricky operations.
+
+If possible, use the same example snippets that your in-code documentation uses. For example, use the snippets in your `examples.py` that Sphinx ingests via its [literalinclude](https://www.sphinx-doc.org/en/1.5/markup/code.html?highlight=code%20examples#includes) directive. The `examples.py` file containing the snippets should reside alongside your package's code, and should be tested in an automated fashion.
+
+Each example in the *Examples* section starts with an H3 that describes the example. At the top of this section, just under the *Examples* H2, add a bulleted list linking to each example H3. Each example should deep-link to the types and/or members used in the example.
+
+* [Create the thing](#create-the-thing)
```
Comment (prmathur-microsoft) [inline sdk/iot/Azure.Iot.Hub.Service/README.md:53]: I don't recall us having the 'thing' in the SDK :)

File: sdk/iot/Azure.Iot.Hub.Service/src/Customized/Models/DigitalTwinInterfaces.cs [hunk 2]
```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Iot.Hub.Service.Models
+{
+    /// <summary>
+    /// The DigitalTwinInterfaces.
+    /// </summary>
```
Comment (prmathur-microsoft) [inline sdk/iot/Azure.Iot.Hub.Service/src/Customized/Models/DigitalTwinInterfaces.cs:8]: Can we mark a todo to check on the name of this?

File: sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice.json [hunk 3]
```diff
@@ -0,0 +1,3093 @@
+﻿{
+  "swagger": "2.0",
+  "info": {
+    "version": "2020-03-13",
+    "title": "IotHub Gateway Service APIs"
```
Comment (prmathur-microsoft) [inline sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice.json:5]: Is there a new swagger? doesn't the branch already contain one?  I see one here - https://github.com/Azure/azure-sdk-for-net/blob/feature/IoT-Hub/sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice.json

**Review by prmathur-microsoft — COMMENTED** [review]
CIL

**Comment by prmathur-microsoft** [comment]
There is a conflict for this branch. Might resolve my comments post that. Please check.

**Review by prmathur-microsoft — APPROVED** [review]

**Comment by barustum** [comment]
@prmathur-microsoft The comments you made are on files that I haven't touched. I only touched the API Design.md file. I have yet to merge from feature/IoT-Hub branch. I am working on that now. I set the PR to draft until I resolve the merge conflicts.

**Comment by barustum** [comment]
Abandoning this PR as I haven't merged from the feature branch into my private branch in a while. Created a new PR:
https://github.com/Azure/azure-sdk-for-net/pull/12728

**Comment by barustum** [comment]
Abandoning this PR as I haven't merged from the feature branch into my private branch in a while. Created a new PR:
#12728

**Pull request closed without merging**